Formula/ldid.rb

```diff
@@ -17,7 +17,7 @@ class Ldid < Formula
   depends_on "openssl"
 
   def install
-    inreplace "./make.sh", /^.*\/Applications\/Xcode-5.1.1.app.*/, ""
+    inreplace "./make.sh", %r{^.*\/Applications\/Xcode-5.1.1.app.*}, ""
```

Comment:
Not a huge deal, but you don't need to escape forward slashes inside a `%r{}` regex.